extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -126,7 +126,8 @@ public function getWikiVariables() {
 			'siteMessage' => $this->getSiteMessage(),
 			'siteName' => $wgSitename,
 			'theme' => SassUtil::getOasisSettings(),
-			'wikiCategories' => WikiFactoryHub::getInstance()->getWikiCategoryNames( $wgCityId )
+			'wikiCategories' => WikiFactoryHub::getInstance()->getWikiCategoryNames( $wgCityId ),
+			'homepage' => ( new WikiaLogoHelper() )->getCentralUrlForLang( $wgLang->getCode() )
```

Comment:
Keep the alphabetic order, please.

Maybe we should also add a check for `WikiaLogoHelper` to not break Mercury when this class is missing somehow. How about a new method `$this->getHomepageUrl()`?